src/Components/TagPickerGridComponent/package.json

```diff
@@ -19,7 +19,7 @@
   "devDependencies": {
     "@types/react": "^16.8",
     "@types/react-dom": "^16.8",
-    "pcf-scripts": "^1",
-    "pcf-start": "^1"
+    "pcf-scripts": "~0",
+    "pcf-start": "~0"
```

Comment:
Normal?

Reply:
Somewhat, the project wouldn't build without doing this. We'll be doing updates to dependencies after this PR anyways.